synchronizer/synchronizer.go

```diff
@@ -562,12 +563,18 @@ func (s *ClientSynchronizer) RequestAndProcessRollupGenesisBlock(dbTx pgx.Tx, la
 		log.Error("error getting rollupInfoByBlockRange after set the genesis: ", err)
 		return err
 	}
-	// Check that the response is the expected. It should be 1 block with 2 orders
+	// Check that the response is the expected. It should be 1 block with ForkID event
+	log.Debugf("genesis block (%d) events: %+v", lastEthBlockSynced.BlockNumber, order)
 	err = sanityCheckForGenesisBlockRollupInfo(blocks, order)
 	if err != nil {
 		return err
 	}
-	forkId := s.state.GetForkIDByBlockNumber(blocks[0].BlockNumber)
+	log.Infof("Processing genesis block %d orders: %+v", lastEthBlockSynced.BlockNumber, order)
```

Comment:
Do we want to keep this log, since we have the similar debug log a couple of lines before? Maybe consider putting that one to info... WDYT?

Reply:
I change the previous one, need to specify that is before sanitycheck and the info one is for real process